### PR TITLE
Replace stored credentials with persona test user accounts

### PR DESCRIPTION
**Pull request by davehunt**
I was working on something else, and noticed that this tests repo is relying on static credentials when it could just use persona test user. Pinging @m8ttyB for review.

### CODE REVIEW

**Comment by m8ttyB** [comment]
@davehunt Good catch .. great even! The long term goal was to move make a full move to personatestuser.org accounts.

We did the refactor and cleanup work to do this but never the final trigger pull to make it happen. The project has been in a state of limbo with the plan to sunset Affliates in favor of a different solution. Hence we stopped maintaining the test suite.

Great-fully, since you've done the work, I'll r? and merge.

File: pages/base.py [hunk 1]
```diff
@@ -43,16 +43,29 @@ def is_user_logged_in(self):
...
-    def get_new_persona_credentials(self):
-        url = "http://personatestuser.org/email/"
-        response = urllib2.urlopen(url).read()
-        decode = json.loads(response)
-        credentials = {
-            'email': decode['email'],
-            'password': decode['pass']
-        }
+    def _create_persona_test_user(self):
+        response = urllib2.urlopen('http://personatestuser.org/email/').read()
```
Comment (m8ttyB) [inline pages/base.py]: Nice, `.read()`

File: pages/base.py [hunk 2]
```diff
@@ -43,16 +43,29 @@ def is_user_logged_in(self):
...
+            lambda s: self.is_user_logged_in)
+        from pages.home import Home
+        return Home(self.testsetup)
+
+    def click_login(self):
+        self.selenium.find_element(*self._login_browser_id_locator).click()
+
+    @property
+    def header(self):
+        return self.selenium.find_element(*self._page_header_locator).text.replace('\n', ' ')
```
Comment (m8ttyB) [inline pages/base.py]: ++

**Comment by m8ttyB** [comment]
r+ and merging. Thanks @davehunt ! Splendid work.

**Comment by davehunt** [comment]
Thanks @m8ttyB I was actually using affiliates locally to test changes to pytest-mozwebqa around how it handles credentials, and I noticed that it doesn't really need them, so thought I'd do away with them!

**Comment by m8ttyB** [comment]
Looks like the coffin nails have been lined up and will start being put in the lid https://blog.mozilla.org/community/2015/05/12/shutting-down-firefox-affiliates/

**Pull request merged**